plasmapy/formulary/collisions.py

```diff
@@ -2300,6 +2321,93 @@ def Lorentz_collision_frequency(self):
             / (self.test_particle.mass**2 * self.v_a**3)
         ).to(u.Hz)
 
+    @cached_property
+    def v_e(self):
+        r"""Average momentum relaxation rate for a slowly flowing Maxwellian
```

Comment:
I think it's important to state that "flowing" is here relative to another population of stationary ions (and therefore this is electron-ion collisions, which are dominant usually). 

Also, we should enforce the meeting of 'slowly' here with a physics error. Callen defines a parameter V_{Tss'} (Eq. 2.113) that  |Vs - Vs'| << V_{Tss'}, so we should raise a physics error if that is not true. 

Also, this whole model might only make sense when you have a drifting Maxwellian, e.g. you use `v_a` as input not `T_a`. I wonder if we should store T_a and v_a (as entered, so one as None) so that in this case we can raise an error if T_a was used. Elsewhere we can make _T_a  and _v_a private properties that do the conversion where allowable (like in the main collision frequencies)?

Reply:
Sure, it's definitely a good idea to enforce that definition

For calculating v_a, is this acceptable? 
v_a = (2 * k_B * self.T_a / self.test_particle.mass)**0.5

Reply:
So, in the general collision frequencies yes, but that only really makes sense when the two populations have no bulk motion relative to one another (so their relative velocity is just the thermal velocity). In this case they are explicitly flowing relative to one another, so I don't think it makes sense to calculate v_a from T_a, because the bulk flow velocity is totally different from the thermal velocity. It's like the thermal velocity is the movement in the plasma CM frame, but then it can also be drifting. Does that make sense?

Reply:
Don't we need T_a for calculating V_Tss' ?

Reply:
Yes, we do, so we would need v_a and T_a in this case (because they're not the same thing if v_a is not just the thermal velocity associated with T_a)

I think we should allow the user to set either or both?